Maple2.Database/Model/Quest.cs

```diff
@@ -1,47 +1,87 @@
 ﻿using System.Collections.Generic;
 using System.Diagnostics.CodeAnalysis;
+using Maple2.Database.Extensions;
 using Maple2.Model.Enum;
+using Maple2.Model.Metadata;
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
 
 namespace Maple2.Database.Model;
 
 public class Quest {
     public int Id { get; set; }
+    public long OwnerId { get; set; }
```

Comment:
I assume this means there are both Account+Character quests?

Reply:
Correct